components/VotesTable/VotesTableRow.tsx

```diff
@@ -29,6 +29,7 @@ export function VotesTableRow({
   const { signer } = useWalletContext();
 
   const { decodedIdentifier, title, origin, options, isCommitted, isRevealed, decryptedVote, correctVote } = vote;
+  const maxDecimals = getPrecisionForIdentifier(decodedIdentifier);
```

Comment:
Use the decimal precision for the identifier as the max decimals (this can be different than 18 for some)